packages/framework/src/Model/Statistics/StatisticsRepository.php

```diff
@@ -83,4 +86,76 @@ function (array $item) {
             $query->getResult()
         );
     }
+
+    /**
+     * @param \DateTimeImmutable $startDateTime
+     * @param \DateTimeImmutable $endDateTime
+     * @return int
+     */
+    public function getNewCustomersCountBetweenDates(DateTimeImmutable $startDateTime, DateTimeImmutable $endDateTime)
```

Comment:
dont you want to use here strict-types in new methods and parts ??

Reply:
I added strict types